src/daemon/gateway.go

```diff
@@ -183,14 +184,99 @@ func (gw *Gateway) GetBlockBySeq(seq uint64) (block coin.SignedBlock, ok bool) {
 	return
 }
 
+// GetInputData returns the data of an inputs
```

Comment:
`GetInputData`, `GetTransactionInputsData`, `GetBlockInputsData`, `GetSignedBlockInputsData` y `GetSignedBlocksInputsData` were added as a centralized way to access the transactions inputs data.